src/sentry/utils/env.py

```diff
@@ -35,6 +36,30 @@ def gcp_project_id() -> str:
     return "__unknown_gcp_project__"
 
 
+# TODO(getsentry/team-ospo#190): Remove once fully deployed.
+def log_gcp_credentials_details(logger) -> None:
+    credentials, project_id = default()
+
+    if not credentials:
+        logger.error("gcp.credentials.notfound")
+
+    logger.info(
```

Comment:
add a else statement? other wise if no credentials it will still try to access its attributes.

also instead of using `.` accessor, ex: `credentials.client_id`, use python's has_attribute so you'll be safe in case its looking for an invalid attribute

Reply:
Done.